common/autoconf/platform.m4

```diff
@@ -381,6 +381,7 @@ AC_DEFUN([PLATFORM_SETUP_LEGACY_VARS],
 
   # ZERO_ARCHDEF is used to enable architecture-specific code
   case "${OPENJDK_TARGET_CPU}" in
+    loongarch64)  ZERO_ARCHDEF=LOONGARCH64 ;;
```

Comment:
This is not necessary, line 390 already does that.

Reply:
But the line 390 is for x86, doesn't it? Please point out my fault!

Thanks,
Leslie Zhai

Reply:
The other 390 ;-) 391 after the patch.

Reply:
The `*)` case.